### PR TITLE
Actions: Automatically add language-specific labels to pull requests.

### PR DESCRIPTION
**Pull request by tausbn**
This seems like a cool feature, so I went ahead and created a configuration file for all of the languages in this repo. Also adds the `documentation` label to all PRs that modify `.qhelp` and `.md` files.
See more about this feature here:

https://github.com/actions/labeler/blob/master/README.md

(Ping @hvitved @jbj @aschackmull @max-schaefer @felicitymay)

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - javascript/**/*
+  - change-notes/**/*javascript*
+
+Python:
+  - python/**/*
+  - change-notes/**/*python*
+
+documentation:
+  - **/*.qhelp
+  - **/*.md
```
Comment (felicitymay) [inline .github/labeler.yml]: Sounds good to me, but please also include: `docs/***/*` in this category.

Comment (tausbn) [reply .github/labeler.yml]: Done! Thanks for the suggestion.

**Comment by aschackmull** [comment]
Something appears to not be working:
```
##[error]YAMLException: unidentified alias "*/*.qhelp" at line 22, column 15:
      - **/*.qhelp
                  ^
##[error]unidentified alias "*/*.qhelp" at line 22, column 15:
      - **/*.qhelp
                  ^
##[error]Node run failed with exit code 1
```

**Comment by jbj** [comment]
That got fixed in #2527, but the check is still failing.

**Comment by RasmusWL** [comment]
yes, since it will use the version of the yml file that is on the PR, it will all work in the GloriusFuture™.

File: .github/labeler.yml (hunk 1 above)
Comment (Hinaroos67) [reply .github/labeler.yml:23]: Hinaroo$6798

**Pull request merged**